zerver/migrations/0032_verify_all_medium_avatar_images.py

```diff
@@ -12,7 +12,7 @@ def verify_medium_avatar_image(apps, schema_editor):
     # type: (StateApps, DatabaseSchemaEditor) -> None
     user_profile_model = apps.get_model('zerver', 'UserProfile')
     for user_profile in user_profile_model.objects.filter(avatar_source=u"U"):
-        upload_backend.ensure_medium_avatar_image(user_profile.email)
+        upload_backend.ensure_medium_avatar_image(user_profile.id)
```

Comment:
yes it does makes sense. I actually didn't really thought of having email let the way it is and have ensure_medium_avatar_image call user_avatar_hash with user_profile. I mean user_avatar_path now.